initrd/etc/functions

```diff
@@ -697,8 +697,17 @@ mount_possible_boot_device() {
 		return 1
 	fi
 
-	TRACE "Try mounting $BOOT_DEV as /boot"
-	if mount -o ro "$BOOT_DEV" /boot >/dev/null 2>&1; then
+	# Get the size of BOOT_DEV in 512-byte sectors
+	sectors=$(blockdev --getsz "$BOOT_DEV")
+
+	# Check if the partition is small (less than 2MB, which is 4096 sectors)
```

Comment:
@JonathonHall-Purism @marmarek : kinda a hack, but works preventing exfat error spurring into dmesg fixing https://github.com/linuxboot/heads/issues/1783

Reply:
alternative solution could be looking for fs signature first (blkid?) but I'm not sure if busybox(?) is capable of that...

Reply:
or explicitly skipping "bios boot" partition (would need to get the partition type - again, not sure if easy in busybox

Reply:
if alternatives are not easy to do, I'm okay with the size check too - 2MB is small enough that no legitimate /boot would be affected.

Reply:
I'm OK with this as a band-aid improvement :+1:   It does suppress some spurious warnings and I don't see any downside.

I've already made multiple changes trying to suppress these annoying exFAT warnings - IMO, I think in the long run we either need to patch out that warning from the kernel or stop dumping kernel warnings to the console.

This is a great improvement but overall this is looking like a whack-a-mole game trying to address each situation individually.

Again though, let's merge this and I'll open a separate issue for the exFAT warnings.

Reply:
Actually, @marmarek, it does look for the partition type above (`is_gpt_bios_grub`) - why didn't that exclude your bios-grub partition?

I still don't have a problem with this sector count check though, it's not going to cause any harm.

Reply:
Opened #1788 for exFAT warnings in general